assignments/006-simple-snakemake-maneuvers/Snakefile

```diff
@@ -1,30 +1,27 @@
 
+# run it over 16 samples
+SAMPLES = ['DPCh_plate1_B10_S22', 'DPCh_plate1_B11_S23', 'DPCh_plate1_B12_S24', 'DPCh_plate1_C10_S34', 'DPCh_plate1_C11_S35', 'DPCh_plate1_C12_S36', 'DPCh_plate1_D09_S45', 'DPCh_plate1_D11_S47', 'DPCh_plate1_F10_S70', 'DPCh_plate1_F11_S71', 'DPCh_plate1_F12_S72', 'DPCh_plate1_G09_S81', 'DPCh_plate1_G10_S82', 'DPCh_plate1_G12_S84', 'DPCh_plate1_H09_S93', 'DPCh_plate1_H10_S94']
 
-# run it over 3 samples
-SAMPLES = ['A', 'B', 'C']
-
-# variant calling over two "chromosomes"
-CHROMOS = [ "NC_037122.1f5t9", "NC_037123.1f10t14"]
-
+# variant calling over four "chromosomes"
+CHROMOS = [ "NC_037122.1f5t9", "NC_037123.1f10t14", "NC_037124.1f8t16", "NC_037125.1f20t24"]
 
 rule genome_faidx:
   input:
-    "resources/genome.fasta",
+    "data/genome/genome.fasta",
   output:
-    "resources/genome.fasta.fai",
+    "results/genome.fasta.fai",
```

Comment:
This actually wants to be data/genome/genome.fasta.fai.  That is where the fai index will go.  It is fine for the dry run, but it will fail on a real run.